### PR TITLE
docs(custom-ca): document mounting a keystore

### PR DESCRIPTION
**Pull request by ethanfrogers**
document an approach for mounting a custom keystore for a Spinnaker
component

### CODE REVIEW

File: _posts/admin/2018-12-19-custom-ca.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+4. Redeploy Spinnaker using `hal deploy apply`.
+
+
+The Spinnaker component for which you configured the volume mount should now be using the new trust/key store by default.
+
+
+***
+
+### More Resources: 
+- [Describe Related Blog Post here](http://blog.armory.io/BLOG-POST-HERE)
```
Comment (german-muzquiz) [inline _posts/admin/2018-12-19-custom-ca.md]: It seems that this section doesn't apply to this article

File: _posts/admin/2018-12-19-custom-ca.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+```
+$ kubectl create secret generic -n {your-spinnaker-namespace} internal-trust-store --from-file /tmp/custom-trust-store/cacerts
+```
+
+3. Configure a Spinnaker service with the new trust/key store using volume mount. In this example we'll be configuring Front50 with this new store.
+
+```
+# ~/.hal/default/service-settings/front50.yml
+kubernetes:
+  volumes:
```
Comment (german-muzquiz) [inline _posts/admin/2018-12-19-custom-ca.md]: I have seen deployment yamls having `volumeMounts` entries to describe where the volume will be mounted, and `volumes` to describe the actual secret name. Just wondering if this configuration is not missing anything.

File: _posts/admin/2018-12-19-custom-ca.md (hunk 1 above)
Comment (ethanfrogers) [reply _posts/admin/2018-12-19-custom-ca.md]: WHOOPS!

File: _posts/admin/2018-12-19-custom-ca.md (hunk 2 above)
Comment (ethanfrogers) [reply _posts/admin/2018-12-19-custom-ca.md]: This configuration isn't missing anything. The difference is a result of how various extension points have been implemented in Halyard which we're attempting to address [here](https://github.com/spinnaker/spinnaker/issues/3743).

**Pull request merged**